sdk/ml/azure-ai-ml/azure/ai/ml/constants/__init__.py

```diff
@@ -8,10 +8,12 @@
 from azure.ai.ml._restclient.v2023_10_01.models import ListViewType
 
 from ._assets import IPProtectionLevel
-from ._common import AssetTypes, InputTypes, InputOutputModes, ModelType, Scope, TimeZone, WorkspaceKind
+from ._common import AssetTypes, InputOutputModes, InputTypes, ModelType, Scope, TimeZone, WorkspaceKind
```

Comment:
is this intended change?

Reply:
This was an automatic change from running precommit hooks I believe